platform/security/certificate/security-certificate-generator/src/main/java/org/codice/ddf/security/certificate/generator/CertificateCommand.java

```diff
@@ -82,12 +142,33 @@ public static String configureDemoCert(String commonName) {
      * private key is also stored in the entry.
      *
      * @param dn String params in the form {@code attrKey=attrVal} composing a distinguished name.
-     *           e.g. {@code configureDemoCertWithDN("cn=John Whorfin", "o=Yoyodyne", "l=San Narciso", "st=California", "c=US")}
+     *           e.g. {@code configureDemoCertWithDN(new String[] {"cn=John Whorfin", "o=Yoyodyne", "l=San Narciso", "st=California", "c=US"), null}
+     * @return the string used as the common name in the new certificate
+     */
+    public static String configureDemoCertWithDN(String[] dn) {
+        return configureDemoCertWithDN(dn, null);
+    }
+
+    /**
+     * Generates new signed certificate. The input parameter is the full set of attributes of the
+     * distinguished name for the cert and optional subject alternative names. It must include a
+     * single {@code CN} value for the common name.
+     * <p>
+     * Postcondition is the server keystore is updated to include a private entry. The private
+     * entry has the new certificate chain that connects the server to the Demo CA. The matching
+     * private key is also stored in the entry.
+     *
+     * @param dn              String params in the form {@code attrKey=attrVal} composing a distinguished name.
+     *                        e.g. {@code configureDemoCertWithDN(new String[] {"cn=John Whorfin", "o=Yoyodyne", "l=San Narciso", "st=California", "c=US"), null}
```

Comment:
I prefer to keep the Nullable and allow either without the creation of an empty array. Or as I said before, I would change this to be an Optional.